pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor_test.go

```diff
@@ -252,10 +297,13 @@ func TestStreamIngestionFrontierProcessor(t *testing.T) {
 
 				var ingestedTimestamp hlc.Timestamp
 				require.NoError(t, protoutil.Unmarshal([]byte(*protoBytes), &ingestedTimestamp))
-				// Ensure that the rows emitted by the frontier never regress the ts.
+				// Ensure that the rows emitted by the frontier never regress the ts or
+				// appear prior to a checkpoint.
```

Comment:
I guess ensures that we start off from the correct checkpoint timestamp?

Do we need explicit test to ensure that when we create frontier processor it's frontier correctly initialized based on the checkpoint?